assets/app/styles/_components.less

```diff
@@ -229,18 +228,43 @@
       border: 1px solid transparent;
       padding: 10px;
       position: relative;
-      margin: 0 10px 10px 0;
+      margin-bottom: 10px;
       text-align: center;
       -webkit-font-smoothing: antialiased;
+
+      + .pod {
+        margin-left: 10px;
+      }
+
       .pod-status-label {
         &:before, &:after {
           font-family: 'PatternFlyIcons-webfont';
           font-weight: normal;
         }
       }
+      &.pod-multiple {
+        position: relative;
+        
+        .fa-ellipsis-h {
+          position: absolute;
+          right: 5px;
+          bottom: 0;
+          font-size: 14px;
+          color: #333;
+        }
```

Comment:
We should add the cursor and outline hover styles. They don't work for collapsed pods at mobile resolutions now since we disable the osc hover.